src/utils/elementfeature_utils.jl

```diff
@@ -123,6 +120,8 @@ function get_bins(
     logspaced::Bool = default_log(feature_name, lookup_table),
     categorical::Bool = default_categorical(feature_name, lookup_table),
 )
+    colnames = names(lookup_table)
+    @assert feature_name in colnames "Your lookup table must have a column with the same name as your feature to be usable!"
```

Comment:
```suggestion
    @assert feature_name in colnames && "Symbol" in colnames "Your lookup table must have a column called :Symbol and one with the same name as your feature ($(feature_name)) to be usable!"
```

Reply:
we don't actually need it in this check though